lib/merchant_parser.rb

```diff
@@ -0,0 +1,24 @@
+require 'CSV'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.